core/src/main/java/org/eclipse/hono/util/TenantConstants.java

```diff
@@ -60,9 +60,9 @@
      */
     public static final String FIELD_PAYLOAD_PUBLIC_KEY = "public-key";
     /**
-     * The name of the property that contains the trusted certificate authority configured for a tenant.
+     * The name of the property that contains the list of trusted certificate authorities configured for a tenant.
      */
-    public static final String FIELD_PAYLOAD_TRUSTED_CA = "trusted-ca";
+    public static final String FIELD_PAYLOAD_TRUST_STORE = "trust-store";
```

Comment:
I wonder if we can keep the original name but change its content model to either support a single JSON *object* or an *array* of objects. That way we could more easily keep backwards compatibility.

Reply:
@sophokles73 how about we then keep the currently available `setTrustAnchor`and `getTrustAnchor`methods? These calls can assume that the `trusted-ca` property contains JSON object.

Reply:
My main concern is backwards compatibility, i.e. it would be nice, if existing database content would not need to be migrated, i.e. it should be possible to map a JSON structure containing a single trusted ca as a JSON object into the `TenantObject` and then use e.g. *getTrustAnchor` to retrieve the *list* of trust anchors (which in this case would contain a single element only).